src/tests/multihost/alltests/test_inmemory_cache.py

```diff
@@ -0,0 +1,329 @@
+""" Automation of memory cache suite
+
```

Comment:
Please add metadata for polarion so the tests are linked to existing requirement.